styles.css

```diff
@@ -41,6 +41,19 @@ body {
   }
 }
 
+/* *Loading gif* */
+
+#gif-load {
+  display: none;
+  flex-direction: column;
+  align-items: center;
+}
+
+#gif-load img {
+  width: auto;
+  height: 20vw;
```

Comment:
could you make this 10vw? I think it makes a big difference, particularly when we switch to bigger screens

Reply:
reply test

Reply:
changed now